v2/tests/log_sink_console_int/log_sink_console_int.c

```diff
@@ -237,6 +253,23 @@ static void log_sink_console_with_a_context_with_string_works(void)
     LOG_CONTEXT_DESTROY(test_context);
 }
 
+static void log_sink_console_with_a_context_with_boolean_works(void)
+{
+    // arrange
+    LOG_CONTEXT_HANDLE test_context;
+    LOG_CONTEXT_CREATE(test_context, NULL,
+        LOG_CONTEXT_STRING_PROPERTY(my_string, "1 + 1 == 3 is %s (%d)", MU_BOOL_VALUE(0)));
```

Comment:
>1 + 1 == 3 [](http://example.com/codeflow?start=48&length=10)

Can we make this test look a little bit less like a test wanting to exercise int values and rather do something with a bool?

Reply:
modified